cloud/edgecontroller/pkg/cloudhub/common/io/hubio.go

```diff
@@ -0,0 +1,51 @@
+package io
+
+import (
+	"encoding/json"
+	"github.com/gorilla/websocket"
```

Comment:
nit: Please seperate, golang and vendor packages.

Reply:
code updated